# customer message for prediction
#

std_msgs/Header header

std_msgs/Int16[] predicted_center_x
std_msgs/Int16[] predicted_center_y
std_msgs/Float64[] predicted_distance
std_msgs/Float64[] predicted_yaw
